### PR TITLE
redesign: add all dimens

### PR DESCRIPTION
**Pull request by laxyapahuja**
## Description 
I've noticed that there are similarities in dimens additions in #119, #117, #112, #110 etc. To avoid merge conflicts, I've compiled all additions into one PR. The dimens additions from all PRs can be removed after this PR gets merged.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -46,7 +81,20 @@
...
+    <dimen name="layout_corner_radius_50dp">50dp</dimen>
+
+    <dimen name="layout_width_25dp">25dp</dimen>
+    <dimen name="layout_width_105dp">105dp</dimen>
+
+    <dimen name="layout_height_25dp">25dp</dimen>
+    <dimen name="layout_height_38dp">38dp</dimen>
+    <dimen name="layout_peak_height_350">350dp</dimen>
+    <dimen name="layout_height_450dp">450dp</dimen>
 </resources>
```
Comment (rachittt) [inline app/src/main/res/values/dimens.xml]: add a blank line at the end of the file

**Comment by rachittt** [comment]
@luckyman20 all the negative dimensions are used for the bottom margin of CardView. I think it should be uniform for all the cards. Can you tell which dimension out of the three(-50,-40 and -30) should be used?

**Comment by luckyman20** [comment]
@rachittt Can you please post screenshot for all 3? It would help me in accessing it better.

**Comment by rachittt** [comment]
@luckyman20 Check the screenshots in the following PRs:
-30dp -> #112 
-40dp -> #115 
-50dp -> #118 
we are using negative dimension to hide the corner radius of the CardView

**Comment by luckyman20** [comment]
@rachittt `-40dp` looks good to me.

**Pull request merged**